rectools/models/implicit_als.py

```diff
@@ -218,8 +212,237 @@ def _fit_paired_factors(
         features_model.fit(xy_csr)
         x_factors = features_model.user_factors.to_numpy()
     else:
+        features_model_params.update(
+            {
+                "num_threads": model.num_threads,
+                "use_native": model.use_native,
+                "use_cg": model.use_cg,
+            }
+        )
         features_model = CPUAlternatingLeastSquares(**features_model_params)
         features_model.item_factors = y_factors
         features_model.fit(xy_csr)
         x_factors = features_model.user_factors
     return x_factors
+
+
+def _init_latent_factors_cpu(
```

Comment:
Please add to this and to the following function short comments that the logic is copied and pasted from the implicit

Reply:
Done